src/apic-extension/azext_apic_extension/aaz/latest/apic/api/_update.py

```diff
@@ -86,15 +89,14 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema.contacts = AAZListArg(
             options=["--contacts"],
             arg_group="Properties",
-            help="The contact information for the API.",
+            help="The set of contacts",
             nullable=True,
         )
         _args_schema.custom_properties = AAZFreeFormDictArg(
             options=["--custom-properties"],
             arg_group="Properties",
             help="The custom metadata defined for API catalog entities.",
             nullable=True,
-            blank={},
```

Comment:
Expected change. `black={}` will set custom properties' value to `{}` if user does not provide any value to `--custom-properties` parameter. This is not a valid scenario as it causes ambiguity. With this change, Azure CLI will raise errors if user does not provide value for `--custom-properties`.